src/gui/auth/qgsauthmasterpassresetdialog.cpp

```diff
@@ -48,6 +47,14 @@ QgsMasterPasswordResetDialog::QgsMasterPasswordResetDialog( QWidget *parent )
       chkKeepBackup->hide();
     }
   }
+
+  QString warning = tr( "Your authentication database will be duplicated and re-encrypted using the new password." );
```

Comment:
```suggestion
  QString warning = tr( "Your authentication database will be re-encrypted using the new password." );
```

Reply:
or maybe even better to globally rename "authentication database" to "authentication configuration storage"?

Reply:
ok, done